tests/hardhat/Tokens/mintAndRedeemTest.ts

```diff
@@ -231,7 +231,7 @@ describe("VToken", function () {
 
       expect(result) // eslint-disable-line @typescript-eslint/no-floating-promises
         .to.emit(vToken, "Mint")
-        .withArgs(minterAddress, mintAmount, mintTokens);
+        .withArgs(minterAddress, mintAmount, mintTokens, mintAmount);
```

Comment:
Assuming the initial balance of vTokens for the user was 0, the last argument of the event will be `minTokens` (the updated balance of vTokens)